hack/cherry_pick_pull.sh

```diff
@@ -0,0 +1,260 @@
+#!/usr/bin/env bash
+
+# Copyright 2015 The Kubernetes Authors.
```

Comment:
Do we need change the date to 2024?

Reply:
Probably not, it's a full copy from K8s, ref: 
https://github.com/kubeedge/kubeedge/blob/master/hack/cherry_pick_pull.sh